querybook/webapp/components/DataDocDAGExporter/DataDocDAGExporter.tsx

```diff
@@ -39,6 +40,16 @@ export const DataDocDAGExporter: React.FunctionComponent<IProps> = ({
         IDataQueryCell[]
     >([]);
 
+    const deleteGraphQueryCell = React.useCallback(
+        (id: number) => {
+            setGraphQueryCells((cells) => {
+                const currentCells = [...cells];
+                return currentCells.filter((cell) => cell.id !== id);
+            });
+        },
+        [setGraphQueryCells]
```

Comment:
memo for setGraphQueryCells is not required (i dont think there is a linter warning for this)